zh/restore-data-using-tidb-lightning.md

```diff
@@ -79,14 +81,48 @@ tikv-importer 可以在一个现有的 TiDB 集群上启用，或者在新建 Ti
               [s3]
               type = s3
               provider = AWS
-              env_auth = false
+              env_auth = true
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              region = us-east-1
+              [ceph]
+              type = s3
+              provider = Ceph
+              env_auth = true
               access_key_id = <my-access-key>
               secret_access_key = <my-secret-key>
+              endpoint = <ceph-object-store-endpoint>
+              region = :default-placement
+              [gcs]
+              type = google cloud storage
+              # 该服务账号必须被授予 Storage Object Viewer 角色。
+              # 该内容可以通过 `cat <service-account-file.json> | jq -c .` 命令获取。
+              service_account_credentials = <service-account-json-file-content>
+            ```
+
+        + 使用 AWS S3 IAM 绑定 Pod 的授权方式或者 AWS S3 IAM 绑定 ServiceAccount 授权方式时，可以省略 `s3.access_key_id` 以及 `s3.secret_access_key：
```

Comment:
From L65 to L103, it looks you are mixing numbered list with unordered list. Please change the indentation.